index.js

```diff
@@ -133,11 +133,13 @@ var createAssertion = function createAssertion(type, message, wrappers, block, m
 	if (descriptors.length === 0) {
 		throw new RangeError(inspect(type) + ' called with no wrappers defined');
 	}
-	var describeMsgs = [];
-	forEach(wrappers, function (wrapper) {
-		checkThis(wrapper);
-		describeMsgs.push(getThisDescription(wrapper));
-	});
+
+	var describeMsgs = descriptors.reduce(function descriptorReducer(descriptions, descriptor) {
```

Comment:
Using `.reduce` means it requires ES5; my hope is that jest will eventually transpile and work on older engines. I'd prefer to keep using `forEach` here.